src/test/java/com/slack/kaldb/logstore/search/StatsCollectorTest.java

```diff
@@ -1,21 +1,16 @@
 package com.slack.kaldb.logstore.search;
 
-import static com.slack.kaldb.logstore.LuceneIndexStoreImpl.COMMITS_COUNTER;
-import static com.slack.kaldb.logstore.LuceneIndexStoreImpl.MESSAGES_FAILED_COUNTER;
-import static com.slack.kaldb.logstore.LuceneIndexStoreImpl.MESSAGES_RECEIVED_COUNTER;
-import static com.slack.kaldb.logstore.LuceneIndexStoreImpl.REFRESHES_COUNTER;
+import static com.slack.kaldb.logstore.LuceneIndexStoreImpl.*;
```

Comment:
Can we please undo the wildcard imports? We avoid them since they make reading code harder.

Reply:
Yeah, I've fixed this. The default behavior for IntelliJ is to optimize these apparently. 🤔

Reply:
Thought so too. Thanks!